app/views/invoices/show.html.erb

```diff
@@ -3,6 +3,7 @@
   <p>Invoice #:<%= @invoice.id %></p>
   <p>Invoice Status:<%= @invoice.status %></p>
   <p>Invoice created at:<%= @invoice.created_at.strftime("%A, %B %d, %Y") %></p>
+  <p>Total Revenue: <%= number_to_currency(@invoice.total_revenue/100, precision: 0) %></p>
```

Comment:
I'm guilty of putting this bit of logic in the view, not sure if it can be fully classified as presentation logic or not. We'll have to check with Jamison, but I think since you built the total revenue method in the invoice.rb it would be a great place to include the number_to_currency logic.